rc/extra/racer.kak

```diff
@@ -137,9 +136,11 @@ define-command racer-show-doc -docstring "Show the documentation about the rust
         racer_match=$(printf %s\\n "$racer_data" | cut -f1)
         if [ "$racer_match" = "MATCH" ]; then
           racer_doc=$(printf %s\\n "$racer_data" | cut -f9 )
+          remove_surrond_quotes_regex='s/^"\(.*\)"$/\1/g'
+          escape_at_sign_regex="s/@/\\\\@/g"
           racer_doc=$(printf %s\\n "$racer_doc" |
-            sed -e 's/^"\(.*\)"$/\1/g' | # Remove leading and trailing quotes
-            sed -e "s/@/\\\\@/g")        # Escape all @ so that it can be properly used in the string expansion
+            sed -e "$remove_surrond_quotes_regex" \
+                -e "$escape_at_sign_regex")
```

Comment:
This could just be:

    sed -e '
        # Remove leading and trailing quotes
        s/^"\(.*\)"$/\1/g
        # Escape all @ so that it can be properly used in the string expansion
        s/@/\\@/g
    '

A single sed script can contain any number of commands and comments.